src/containers/adminPage/SelectionPanel.jsx

```diff
@@ -56,7 +56,15 @@ function SelectionPanel(props) {
                                                 id={id}
                                                 className="entur-radio-checkbox--round"
                                                 checked={isChecked}
-                                                onChange={() => updateHiddenList(id, 'stops')}
+                                                onChange={() => {
+                                                    let stopUnchecked = false
+                                                    for (stop in stops){
+                                                        if (!document.getElementById(stops[stop].id).checked){stopUnchecked = true}
```

Comment:
Instead of using `document.getElementById`, I think you can use the `onCheck` function as on line 51.